src/renderer/services/const.ts

```diff
@@ -1,9 +1,9 @@
 import { Network } from '../../shared/api/types'
-import { loadEnabledChains } from '../helpers/envHelper'
+import { envOrDefault } from '../helpers/envHelper'
 
 export const DEFAULT_NETWORK: Network = 'testnet'
 export const AVAILABLE_NETWORKS: Network[] = ['testnet', 'chaosnet', 'mainnet']
-export const ENABLED_CHAINS = loadEnabledChains(process.env.REACT_APP_CHAINS_ENABLED)
+export const ENABLED_CHAINS = envOrDefault(process.env.REACT_APP_CHAINS_ENABLED, 'THOR,BNB,BTC').split(',')
```

Comment:
Pls add `process.env.REACT_APP_CHAINS_ENABLED` to env.sample with a comment: 

```
Add chains you want to enable in development mode.
```

Also a similar note to README (in chapter [Development](https://github.com/thorchain/asgardex-electron#development)):

## How to enable (more) chains for development

Add chains you want to have in development to `REACT_APP_CHAINS_ENABLED`, for example

```bash
REACT_APP_CHAINS_ENABLED=THOR,BNB,BTC,ETH,LTC
```